src/components/notification/NotificationCard.tsx

```diff
@@ -6,23 +6,28 @@ import Animated, { runOnJS, useAnimatedStyle, useSharedValue, withTiming } from
 import AntDesignIcon from 'react-native-vector-icons/AntDesign'
 
 import { useHandler } from '../../hooks/useHandler'
+import { getThemedIconUri } from '../../util/CdnUris'
 import { styled } from '../hoc/styled'
 import { Theme, useTheme } from '../services/ThemeContext'
 import { EdgeText } from '../themed/EdgeText'
 
 interface Props {
-  iconUri: string
-  title: string
   message: string
+  title: string
+  type: 'warning' | 'info'
+  iconUri?: string
+
   onPress: () => void | Promise<void>
   onClose?: () => void | Promise<void>
 }
 
 const NotificationCardComponent = (props: Props) => {
-  const { iconUri, title, message, onClose, onPress } = props
   const theme = useTheme()
   const styles = getStyles(theme)
 
+  const { title, type, message, onClose, onPress } = props
+  const { iconUri = type === 'warning' ? getThemedIconUri(theme, 'notifications/icon-warning') : getThemedIconUri(theme, 'notifications/icon-info') } = props
```

Comment:
Please get in touch with Sam about his new icon system - I think this will fit in nicely with what he's working on.

No changes are needed for now, especially since this is a 3.23 cherry pick, but I think there is a chance for alignment here.